client/css/style.css

```diff
@@ -787,8 +787,6 @@ kbd {
 	right: 0;
 	width: 180px;
 	transition: all .4s;
-	-webkit-transform: translateZ(0);
-	transform: translateZ(0);
```

Comment:
It's really strange, but sidebar rendering was fixed by one of the recent commits, so it's not necessary to remove these lines anymore. Magic! Although there is a chance that this problem will appear again in future.